README.md

```diff
@@ -1,678 +1,27 @@
-         *
-         * Each parameter corresponds to a property on this class.  If left unspecified, the copy will have
-         * the same value as the original.
-         */
-        fun copy(
-            first: SymbolPrimitive = this.first,
-            last: SymbolPrimitive = this.last,
-            children: List<Person> = this.children,
-            metas: MetaContainer = this.metas
-        ) {
-            /* ... */
-        }
-
-        /**
-         * Determines if this [Person] is equivalent [other].
-         *
-         * To be equivalent, [other] must be an instance of [Person] and all of its properties (except [metas]) must be
-         * equivalent.
-         *
-         * This is recursive and applies deeply to all child nodes.
-         *
-         */
-        override fun equals(other: Any?): Boolean {
-            /* ... */
-        }
-
-        /**
-         * Computes a hash code for the current instance of [Person].
-         *
-         * The has code is computed once, the first time this function is invoked and the value is then re-used.
-         *
-         * The [metas] property is not an input into the hash code computation.
-         */
-        override fun hashCode(): Int {
-            /* ... */
-        }
-    }
-
-    // ... other stuff
-}
-```
-
-The first thing the reader will note is that `Person` resides within the `SampleTypeDomain` class which is being used as
-a namespace. Some projects have many type domains sharing the same class names and this helps avoid ambiguity and
-namespace pollution.
-
-The next thing the reader might notice is that `Person` implements some functionality provided by Kotlin
-[data classes](https://kotlinlang.org/docs/data-classes.html). Such as `.copy`, `.equals` and `.hashCode`.
-The `data class` feature of Kotlin can't be used in this case because data classes always include all of their
-properties in the `.hashCode()` and `.equals()` implementations, and we do not include [metas] in the definition of
-equivalence for any PIG-generated class. (More on metas later.)
-
-#### Generated Builders
-
-Although the constructor of `Person` seen in the previous section was public, this is not the preferred way of creating
-instances. The preferred way involves using `SampleTypeDomain.Builder`. The following code creates a node representing a
-person and two children:
-
-```Kotlin
-val person = SampleTypeDomain.build {
-    person(
-        "Billy", "Smith",
-        listOf(
-            person("Jack", "Smith", listOf()),
-            person("Sue", "Smith", listOf())
-        )
-    )
-}
-```
+[![PIG Generator](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-ir-generator/badge.svg?)](https://search.maven.org/artifact/org.partiql/partiql-ir-generator)
+[![PIG Runtime](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-ir-generator-runtime/badge.svg?)](https://search.maven.org/artifact/org.partiql/partiql-ir-generator-runtime)
```

Comment:
(non-blocking): Can also add another badge for the GitHub Actions workflow status and the license

```suggestion
[![PIG Runtime](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-ir-generator-runtime/badge.svg?)](https://search.maven.org/artifact/org.partiql/partiql-ir-generator-runtime)
[![License](https://img.shields.io/hexpm/l/plug.svg)](https://github.com/partiql/partiql-ir-generator/blob/main/LICENSE)
[![CI Build](https://github.com/partiql/partiql-ir-generator/actions/workflows/build.yml/badge.svg)](https://github.com/partiql/partiql-ir-generator/actions?query=workflow%3A%22Build+and+run+tests%22)
```